scripts/backend.py

```diff
@@ -104,7 +104,7 @@ def install() -> None:
     Downloads & installs all dependencies for the backend.
     """
     pipenv.create()
-    sys_calls.run(['pipenv', 'install'])
+    sys_calls.run(['pipenv', 'install', '--ignore-pipfile'])
```

Comment:
This change makes it so that we no longer try to upgrade dependencies when installing the app. Instead, it will copy the Pipfile.lock exactly, so everyone will be using the same version of everything.